cmd/monaco/main.go

```diff
@@ -336,3 +344,40 @@ func getDownloadCommand(fileReader util.FileReader) cli.Command {
 	}
 	return command
 }
+func getInitialiseCommand() cli.Command {
+	command := cli.Command{
+		Name:      "init",
+		Usage:     "initialise demo project and folders. Optionally pass a list of APIs to create demo folders or leave blank for all",
+		UsageText: "init [command options]",
+		Before: func(c *cli.Context) error {
+			err := util.SetupLogging(c.Bool("verbose"))
+
+			if err != nil {
+				return err
+			}
+
+			util.Log.Info("Dynatrace Monitoring as Code v" + version.MonitoringAsCode)
+
+			return nil
+		},
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "createSpecificAPI",
+				Usage:   "Comma separated list of API's to create",
+				Aliases: []string{"c"},
```

Comment:
`s` ?

Reply:
`s` is also already reserved :( I think `a` would work?
```
			&cli.StringFlag{
				Name:    "specific-environment",
				Usage:   "Specific environment (from list) to deploy to",
				Aliases: []string{"s"},
			},
```

Reply:
Actually this should not be an issue, as those flags are command scoped.

Reply:
I guess the `continue-on-error` flag doesn't really make sense in the context of `init`.

Reply:
Why `-a`? :thinking: It would be cool to have a list of used flags and their aliases. I believe its confusing to the user if a given alias is used in two commands with two different meanings.

Reply:
But then we are limited to 24 flags :laughing:

Reply:
Actually @Dynatrace-Adam-Gardner the name should be in kebab case (I guess it is called that way :laughing: ) `create-specific-api`.

Reply:
@patrickpichler yes, I know. But that would be consistent 24 flags :smile:. At least we should not use "commonly known flags" for different purposes. E.g.: 

`-v` verbose
`-e` environments
`-p` projects
`-c` config or cluster :thinking: 

ahh, and now I understand @tatomir146:
`-a` for api

Reply:
I like that idea :+1: 
Otherwise we end up in flag hell

Reply:
@didiladi yeah, I think we should maybe use `-api` (not `specific` api). Let's proceed discussion here #182